package.json

```diff
@@ -26,7 +26,6 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.43.5",
-    "react-query": "^3.39.3",
```

Comment:
Are we sure about this?

Reply:
You actually moved the @types there. We need the lib here, and the types might come by default